src/lib/types/events.ts

```diff
@@ -326,6 +328,43 @@ export class FeatureEnvironmentEvent extends BaseEvent {
         this.environment = p.environment;
     }
 }
+export type EnvironmentStrategyExecutionOrder = string[];
+export class FeatureEnvironmentStrategyExecutionOrderUpdatedEvent extends BaseEvent {
```

Comment:
I'd go for something easier to use in a conversation e.g. StrategiesOrderChangedEvent